charts/stable/unifi/SCALE/questions.yaml

```
@@ -532,6 +532,246 @@ questions:
                               type: int
                               min: 9000
                               max: 65535
+        - variable: speedtest
+          label: "Speedtest Service"
+          description: "Speedtest Service"
+          schema:
+            type: dict
+            attrs:
+              - variable: enabled
+                label: "Enable the service"
+                schema:
+                  type: boolean
+                  default: true
+                  hidden: true
```

as these are mostly optional ports, I would prefer them disabled and the option to enable shown:
```suggestion
                  type: boolean
                  default: false
```